scripts/install.sh

```diff
@@ -157,7 +157,8 @@ sudo apt install tpm2-tools
 cd $SCRIPT_DIR/..
 make
 
-FILENAME=`cat /sys/class/net/eno1/address`
+MAC=$(ip addr show | awk '/inet.*brd/{print $NF; exit}')
```

Comment:
Maybe rename the variable, because the command is no the MAC, but the network interface.